src/containers/my-cooperations/cooperation-notes/CooperationNotes.tsx

```diff
@@ -1,219 +1,182 @@
-import { useState, useCallback } from 'react'
+import React, { useState, useCallback, useEffect } from 'react'
```

Comment:
We don't need to import `React`
```suggestion
import { useState, useCallback, useEffect } from 'react'
```